pkg/asset/installconfig/platform.go

```diff
@@ -114,16 +147,26 @@ func (a *Platform) awsPlatform() (*asset.State, error) {
 
 	return assetStateForStringContents(
 		AWSPlatformType,
-		strings.Split(string(region.Contents[0].Data), " ")[0],
+		strings.SplitN(string(region.Contents[0].Data), " ", 2)[0],
```

Comment:
Let's use a transformation (provided by Survey) instead of manipulating the user's choice.

Reply:
> Let's use a transformation (provided by Survey) instead of manipulating the user's choice.

This sounded good, but turned out to be not a big win (I've done it in 362eda8 -> 5b3c205).  It's nice to not have to muck with the response before we return it, but [the validator fires before the tranform][1], so the validator needs to be able to handle both the short and long forms.  I dunno why they didn't transform before validating...

[1]: https://github.com/AlecAivazis/survey/blob/db8e629a54ae1a4442be0deae7be60062c69d111/survey.go#L149-L175